scout/server/blueprints/login/controllers.py

```diff
@@ -104,7 +104,11 @@ def google_login() -> Optional[Response]:
 
     redirect_uri: str = url_for("login.authorized", _external=True)
     try:
-        return oauth_client.google.authorize_redirect(redirect_uri, prompt="select_account")
+        return oauth_client.google.authorize_redirect(
+            redirect_uri,
+            prompt="consent",  # force consent each time (so refresh_token is issued)
+            access_type="offline",  # tells Google to send a refresh token
```

Comment:
This should solve the problem we saw during the test together (Failed to refresh token for provider GOOGLE: invalid_request: Missing required parameter: refresh_token). Now refresh token will be stored in the session and used when token is expired to retrieve a new token to send in chanjo2 requests